src/OI.cpp

```diff
@@ -61,23 +61,23 @@ OI::OI() {
     dButtonR3.reset(new JoystickButton(driver.get(), 10));
     dButtonR3->WhileHeld(new AutonomousCommand());
     dButtonL3.reset(new JoystickButton(driver.get(), 9));
-    dButtonL3->WhenReleased(new ShifterToggle());
+    dButtonL3->WhenPressed(new ShifterToggle());
```

Comment:
Why the switch? What are the cons and pro of using when pressed to when released? Is one better then the other?